### DISCUSSION TITLE
what should i do if my javascript is blocked i think

### DISCUSSION THREAD

**Discussion by 27blao** [3kh0/ext-remover#392] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
a

**Comment by Aka-but-nice** [comment, marked as answer]
To test make a bookmark with javascript:alert('Bookmarks are enabled')

**Reply by bartholemuew** [reply]
this doesn't help

**Reply by Aka-but-nice** [reply]
wut

**Reply by rwg100** [reply]
run the bookmark, if a windows pops up saying "Bookmarks are enabled" then your javascript works

**Reply by 3kh0** [reply]
LOL @bartholemuew use your brain

**Comment by Alex2Mayhem** [comment]
> it doesn't work if you "THINK" javascript is blocked. If nothing pops up when you try the bookmarklet to unblock stuff. It clearly doesn't work.
